scripts/yarn/export.js

```diff
@@ -2,16 +2,75 @@ import { processEnv } from 'env-loader'
 import fs from 'fs'
 import path from 'path'
 import { fileURLToPath } from 'url'
+import Debug from 'debug'
 
-// Clean the export directory to get tugboat working
-if (process.env.BUILD_OPTION === 'static') {
-  const __filename = fileURLToPath(import.meta.url)
-  const __dirname = path.dirname(__filename)
+const exportLog = Debug('next-build:export')
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const logsDir = path.resolve(__dirname, '../../logs')
 
+/**
+ * Pre-build housekeeping
+ */
+if (process.env.BUILD_OPTION === 'static') {
+  // Clean the export directory to get tugboat working
   const exportBuildPath = path.resolve(__dirname, '../../.next/export')
   if (fs.existsSync(exportBuildPath)) {
+    exportLog('Cleaning export directory')
     fs.rmdirSync(exportBuildPath, { recursive: true, force: true })
   }
+
+  // Clean the warning logs
+  if (fs.existsSync(logsDir)) {
+    exportLog('Cleaning logs directory')
+    fs.rmdirSync(logsDir, { recursive: true, force: true })
+  }
 }
 
 await processEnv('next build', true)
+
+/**
+ * Post-build processing
+ */
+if (process.env.BUILD_OPTION === 'static') {
+  // Concatenate all warning-*.md files into a single warnings.md file
+  // Get all warning-*.md files
+  const files = fs
+    .readdirSync(logsDir)
+    .filter((file) => file.startsWith('warning-') && file.endsWith('.md'))
+
+  exportLog(`Found ${files.length} warning files to process`)
+
+  const warnings = []
+
+  for (const file of files) {
+    const filePath = path.join(logsDir, file)
+    const content = fs.readFileSync(filePath, 'utf8')
+    const lines = content.split('\n')
+
+    for (const line of lines) {
+      if (line.trim() === '') continue
+      // Match lines that start with a timestamp
+      // Alternatively, we could write JSON as log lines and parse them...if we
+      // had more metadata. ¯\_(ツ)_/¯
```

Comment:
I could go either way, honestly. If for whatever reason it's malformed JSON, it'll throw an error during parsing, so I kept it like this for now.

One reason we might want to add more metadata to spiffy this up is if we wanted our Slack notification to glob warnings by warning type, sorted by content type. Or something fancy like that. I don't anticipate we'll see a need for this, but it's an option. 🤷